scripts/mailmap-update.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eu
+
+usage () {
+    echo "USAGE: ./update-mailmap.sh <bpftool-repo> <linux-repo>"
```

Comment:
mailmap-update.sh ?

Reply:
Indeed!

Reply:
It even says "bpftool" instead of "libbpf" :facepalm: